newspaper/outputformatters.py

```diff
@@ -60,7 +60,11 @@ def get_formatted(self, top_node):
     def convert_to_text(self):
         txts = []
         for node in list(self.get_top_node()):
-            txt = self.parser.getText(node)
+            try:
+                txt = self.parser.getText(node)
+            except ValueError:  # lxml error
```

Comment:
I think we should at least print out warnings + fail loudly if this happens. Maybe notify the url that failed so the developer can triage further? The ideal is to debug urls that don't work so the library can slowly extract more successfully.